react-native-libraries.json

```diff
@@ -10330,5 +10330,16 @@
     ],
     "ios": true,
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/hirbod/expo-video-metadata",
+    "npmPkg": "expo-video-metadata",
```

Comment:
```suggestion
```

Small nit. Let's skip `npmPkg` since it is not required in your library case - https://github.com/react-native-community/directory?tab=readme-ov-file#npmpkg.